docs/design/windows-design.md

```diff
@@ -120,15 +120,15 @@ We should differentiate the traffic if it is entering OVS from the uplink interf
 table. In encap mode, the packets entering OVS from the uplink interface is output to the bridge interface directly.
 In noEncap mode, there are three kinds of packets entering OVS from the uplink interface:
 
- 1) traffic that is sent to local Pods from Pod on a different Node,
- 2) traffic that is sent to local Pods from a different Node according to the routing configuration,
- 3) traffic on the host network
+ 1) Traffic that is sent to local Pods from Pod on a different Node
+ 2) Traffic that is sent to local Pods from a different Node according to the routing configuration
+ 3) Traffic on the host network
 
 For 1 and 2, the packet enters the OVS pipeline, and the `macRewriteMark` is set to ensure the destination MAC can be
 modified.
 For 3, the packet is output to the OVS bridge interface directly.
 
-The packet is always output to the uplink interface if it is entering OVS from the bridge interface. We
+The packet is always outputted to the uplink interface if it is entering OVS from the bridge interface. We
```

Comment:
both are generally accepted, and I would say output is used much more frequently

Reply:
Thanks. Changed back.